service/src/main/kotlin/fi/espoo/evaka/vasu/Vasu.kt

```diff
@@ -280,10 +280,11 @@ sealed class VasuQuestion(
         val options: List<QuestionOption>,
         val minSelections: Int,
         val maxSelections: Int?,
-        val value: List<String>
+        val value: List<String>,
+        val textValue: Map<String, String>? = emptyMap()
```

Comment:
I think `textValue` field shouldn't be nullable. It just makes using it harder and json deserialization will use the default value if it's missing from the json.